src/controllers/v1.controller.ts

```diff
@@ -173,7 +175,16 @@ export class V1Controller {
 
       // Fetch applications contained in this Load Balancer. Verify they exist and choose
       // one randomly for the relay. First check RPC ID to see if this client should be stuck to an app and node.
-      const { preferredApplicationID, preferredNodeAddress, rpcID } = await this.checkClientStickiness(rawData)
+
+      const { stickiness = false, stickinessDuration = DEFAULT_STICKINESS_DURATION } = loadBalancer
+
+      const { preferredApplicationID, preferredNodeAddress, rpcID } = stickiness
```

Comment:
Looks a bit confusing; stickiness is a boolean. How are we getting these values from it?